### ISSUE TITLE
Typo's in EntityType list of `entity-per-chunk-save-limit`

### ISSUE THREAD

**Issue by bramceulemans** [YouHaveTrouble/minecraft-optimization#48]
As you can see the default `paper.yml` has underscores in the entity names, I think the examples in `README.md` contains typo's because of this.

```diff
     entity-per-chunk-save-limit:
-      fireball: -1
-      small_fireball: -1
-      experience_orb: -1
-      snowball: -1
-      ender_pearl: -1
-      arrow: -1
+      arrow: 16
+      dragonfireball: 3
+      egg: 8
+      ender_pearl: 128
+      experience_orb: 16
+      fireball: 8
+      firework: 8
+      largefireball: 8
+      lingeringpotion: 8
+      llamaspit: 3
+      shulkerbullet: 8
+      sizedfireball: 8
+      snowball: 8
+      spectralarrow: 16
+      splashpotion: 3
+      thrownexpbottle: 3
+      trident: 16
+      witherskull: 4
+      wither_skull: 4
```

**Comment by YouHaveTrouble** [comment]
You might be correct here, as afaik paper just compares the values to EntityType enum, will correct the names soonish

**Comment by bramceulemans** [comment]
Thanks for the super quick response, in case you need the enum, I can post it here!

**Comment by YouHaveTrouble** [comment]
I'm good, just need to fire up text editor and go through it

**Comment by YouHaveTrouble** [comment]
fixed in 32e60b2

**Comment by bramceulemans** [comment]
Thanks a ton!
